tests/llm/test_hugging_face_hub.py

```diff
@@ -0,0 +1,47 @@
+import importlib
+import os
+import unittest
+from unittest.mock import patch
+
+from embedchain.config import BaseLlmConfig
+from embedchain.llm.hugging_face_hub import HuggingFaceHubLlm
+
+
+class TestHuggingFaceHubLlm(unittest.TestCase):
+    def setUp(self):
+        os.environ["HUGGINGFACEHUB_API_TOKEN"] = "test_api_key"
+        self.config = BaseLlmConfig(model="google/flan-t5-xxl", max_tokens=50, temperature=0.7, top_p=0.8)
+
+    def test_init_raises_value_error_without_api_key(self):
+        os.environ.pop("HUGGINGFACEHUB_API_TOKEN")
+        with self.assertRaises(ValueError):
+            HuggingFaceHubLlm()
+
+    def test_get_llm_model_answer_raises_value_error_for_system_prompt(self):
+        llm = HuggingFaceHubLlm(self.config)
```

Comment:
I believe these tests will only pass if we supply the `HUGGINGFACEHUB_ACCESS_TOKEN` env variable. Can you please mock the  actual hugging face object that you are using in `HuggingFaceHubLlm` code?

Reply:
Thanks a lot for the feedback @deshraj! I've added a test mocking the actual HuggingFace object. Please let me know if this aligns with the intended result.